bitacross-worker/docker/multiworker-docker-compose.yml

```diff
@@ -131,6 +131,7 @@ services:
       - SORA_QUIZ_MASTER_ID=SORA_QUIZ_MASTER_ID
       - SORA_QUIZ_ATTENDEE_ID=SORA_QUIZ_ATTENDEE_ID
       - MORALIS_API_URL=http://localhost:19527
+      - KARAT_DAO_API_URL=http://localhost:19527/karat_dao/
```

Comment:
There are no dataproviders in bitacross so there is no need to add this value here.
I'll clean up compose files for bitacross to remove all unnecessary env values.